### ISSUE TITLE
Some Catalouge entries still showing old properties no in the model anymore

### ISSUE THREAD

**Issue by bgajdero** [csse-uoft/ckanext-udc#89]
This record still showing FAIR principles
https://data.urbandatacentre.ca/catalogue/income-support-caseloads

Only visible in View mode. Not visible in Edit mode.
<img width="765" alt="Screenshot 2024-11-22 at 10 49 22" src="https://github.com/user-attachments/assets/82f37be3-73df-4b7d-a582-fc14d113fe63">

**Comment by LesterLyu** [comment]
This happened before when the solr index is not in sync with the CKAN db.

**Comment by bgajdero** [comment]
Do we need a Solr reindex  then?
How did this one not get reindexed?

**Comment by LesterLyu** [comment]
Yes, a solr reindex fixes the issue but sometimes after editing it manually/or doing an API import it shows up again, need more investigation.
Or force the interface to only display the item in the model.

**Comment by bgajdero** [comment]
> Or force the interface to only display the item in the model.
In addition to reindexing, hiding properties, not in the model is absolutely the case.

**Comment by LesterLyu** [comment]
Go to the "Edit" page of the catalogue entry and directly "Update" without any changes can fix the catalogue entry, this let CKAN to update the solr index on that specific catalogue entry.
